core/genesis.go

```diff
@@ -385,7 +385,7 @@ func DefaultRopstenGenesisBlock() *Genesis {
 		Knot:       ReadKnot("./core/knot/ropsten_knot.rlp"),
 		Nonce:      66,
 		ExtraData:  hexutil.MustDecode("0x3535353535353535353535353535353535353535353535353535353535353535"),
-		GasLimit:   []uint64{16777216, 16777216, 16777216},
+		GasLimit:   []uint64{100000, 100000, 100000},
```

Comment:
Could you provide some explanation how you chose these new values for gas limits & difficulties? Are these numbers "final" or just here for testing? Maybe just put some notes in the commit message.

Reply:
This was done just for testing purpose. Adding a note in commit message.